core/arch/arm/kernel/generic_entry_a32.S

```diff
@@ -425,6 +445,9 @@ UNWIND(	.cantunwind)
 
 	bl	plat_cpu_reset_late
 
+    /* this core just comes out of reset, discard undefined l1 contents */
+	bl	arm_cl1_d_invbysetway
```

Comment:
I refactored all changes into three independent CFG items:
CFG_TEE_VXWORKS_SMC: this enables VxWorks extended SMC handling in OP-TEE and corresponding plat-xxxx code.
CFG_TEE_BUILT_IN_ARGS: this enables OP-TEE to used built-in arguments instead of relying on arguments passed in from firmware.
CFG_TEE_SECONARY_ENTRY_L1_INV: this enables OP-TEE's generic boot process to invalidate L1 on secondary entry, necessary for platforms that do not have el3 firmware to take care of this.

Now there are just three independent CFGs and CFG_TEE_BUILT_IN_ARGS and CFG_TEE_SECONDARY_ENTRY_L1_INV can be used in cases other than VxWorks Support.

There is no long overall "VxWorks support" option, instead, users are required to enable VxWorks SMC Ext, Built-in ARGS and SMP L1 Inv to support VxWokrs. We'll document this in Wind River's User Guide.

All above options are covered in mk/config.mk.